### ISSUE TITLE
Throw exception when doesn't have any image

### ISSUE THREAD

**Issue by meysampg** [fandoghpaas/fandogh-cli#54]
### Reproduce step:
1. run a fresh fandogh
2. login
3. run `fandogh image logs`
4. on input, enter `1`

### log:
```
#~ fandogh image logs  
image version: 1
Traceback (most recent call last):
  File "/usr/bin/fandogh", line 11, in <module>
    sys.exit(base())
  File "/usr/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.6/site-packages/fandogh_cli/base_commands.py", line 31, in invoke
    raise exp
  File "/usr/lib/python3.6/site-packages/fandogh_cli/base_commands.py", line 20, in invoke
    return super(FandoghCommand, self).invoke(ctx)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/fandogh_cli/image_commands.py", line 83, in logs
    show_image_logs(image, version)
  File "/usr/lib/python3.6/site-packages/fandogh_cli/image_commands.py", line 67, in show_image_logs
    response = get_image_build(image_name, version, token)
  File "/usr/lib/python3.6/site-packages/fandogh_cli/fandogh_client/__init__.py", line 85, in get_image_build
    response = requests.get(base_images_url + '/' + image_name + '/versions/' + version + '/builds',
TypeError: must be str, not NoneType
```

**Comment by meysampg** [comment]
It doesn't happen if already `init` command of image is run.

**Comment by s-soroosh** [comment]
Hi @meysampg, 

Thank you for reporting issuing this bug. 
That's right, we should not assume that `--image` is optional if the current working directory does not contain `.fandogh/config` file. 

Will addresss it in the next release.

**Comment by s-soroosh** [comment]
@meysampg The bug fixed in the new version `0.3.2`. 

Please don't hesitate open issue if you see any room to improve CLI/Docs.
